app/controllers/providers/show/edit.js

```diff
@@ -448,19 +448,32 @@ export default Controller.extend({
         if (contact.email === m.get('billingContact.email')) {
           roleName.push('billing');
         }
-        if (contact.email === m.get('secondaryBillingContact.email')) {
+        if (contact.eail === m.get('secondaryBillingContact.email')) {
```

Comment:
Is this a typo?

Reply:
`contact.eail`

Reply:
It is.  Good catch. Fixed with new vercel deployment at: https://bracco-esaolore6-datacite.vercel.app/ (2023/03/03)